src/SonarScanner.MSBuild.Common/OperatingSystemProvider.cs

```diff
@@ -53,17 +53,38 @@ public virtual bool DirectoryExists(string path) =>
     public virtual PlatformOS OperatingSystem() =>
         operatingSystem.Value;
 
-    public bool IsAlpine() =>
-        IsAlpineRelease("/etc/os-release")
-        || IsAlpineRelease("/usr/lib/os-release");
-
-    // Not stable testable
-    [ExcludeFromCodeCoverage]
     public virtual bool IsUnix() =>
         OperatingSystem() is PlatformOS.Linux or PlatformOS.Alpine or PlatformOS.MacOSX;
 
-    // Not stable testable, manual testing was done by running the scanner on Windows, Mac OS X and Linux.
-    [ExcludeFromCodeCoverage]
+    [ExcludeFromCodeCoverage]   // We only collect coverage from Windows UTs.
+    public virtual void Set(string destinationPath, int mode)
```

Comment:
This got lost somewhere from the previous comment
```suggestion
    public virtual void SetPermissions(string destinationPath, int mode)
```
or
```suggestion
    public virtual void SetFilePermissions(string destinationPath, int mode)
```

Reply:
Don't forget to update the UT name too